data/transactions/test_files.txt

```diff
@@ -16,10 +18,13 @@ pin-mongos.json
 read-concern.json
 read-pref.json
 reads.json
+retryable-abort-errorLabels.json
 retryable-abort.json
+retryable-commit-errorLabels.json
 retryable-commit.json
 retryable-writes.json
 run-command.json
 transaction-options.json
+transaction-options-repl.json
```

Comment:
Ah, nice catch! Looks like this test was missed from before.

One day we should remove these test_files.txt, and just glob the entire directory.

Reply:
I agree. We have a ticket for it here: https://jira.mongodb.org/browse/CXX-1917